python_modules/dagster/dagster_tests/cli_tests/command_tests/test_cli_commands.py

```diff
@@ -563,3 +565,10 @@ def test_run_wipe_incorrect_delete_message():
     result = runner.invoke(run_wipe_command, input="WRONG\n")
     assert "Exiting without deleting all run history and event logs" in result.output
     assert result.exit_code == 0
+
+
+def test_use_env_vars_for_cli_option():
+    runner = CliRunner(env={"DAGSTER_VERSION": "1"})
+    result = runner.invoke(cli, ["debug"], auto_envvar_prefix="DAGSTER")
```

Comment:
This test if a bit hacky. But it may cause us to keep changing this test if it relies on other commands because the project is evolving so quickly.

Reply:
Our CLIs are protected from breaking changes according to semvar, so it's actually fine if these tests are against another command. Maybe the command listed in https://github.com/dagster-io/dagster/issues/3000 is a good candidate.

Reply:
hmm. it will not work because `dagster api grpc` is a long running process, and the test will hang because of that.

Reply:
Changed to `dagster asset wipe`